documentation/README.md

```diff
@@ -0,0 +1,23 @@
+Ensure that requirements in requirements_dev are full filled.
+
+Make changes to conf.py as per the requirements.
+
+To initialize the Sphinx source directory execute the following inside documentation:
+
+sphinx-quickstart
+
+Currently the directory has been initialized. 
+
+Use sphinx-apidoc to generate reStructuredText files from source code by executing following command:
+
+sphinx-apidoc -f -o <path-to-output> <path-to-module>
+
+Add the generated .rst file into index.rst toctree
```

Comment:
Manually copy? What is required here?

Reply:
@ta-oliver yes you need to manually copy the name of created .rst file to index.rst toc tree.

Reply:
Could we just supply the desired location as the <path-to-output> argument?